File: randcs-infrastructure/src/main/java/org/example/RandcsApplication.java
package org.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class RandcsApplication {
    public static void main(String[] args) {
        SpringApplication.run(RandcsApplication.class, args);
    }
}

File: randcs-infrastructure/src/main/kotlin/org/example/RandcsApplication.kt
package org.example

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
open class RandcsApplication

fun main(args: Array<String>) {
    runApplication<RandcsApplication>(*args)
}